src/catchup/DownloadAndVerifyLedgersWork.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#pragma once
+
+#include "ledger/CheckpointRange.h"
+#include "work/Work.h"
+#include "xdr/Stellar-SCP.h"
+#include "xdr/Stellar-ledger.h"
+
+namespace stellar
+{
+
+class LedgerHeaderHistoryEntry;
+class TmpDir;
+class VerifyLedgerChainWork;
+
+class DownloadAndVerifyLedgersWork : public Work
```

Comment:
rest of the code talks about LedgerChain, so this should be called `DownloadAndVerifyLedgerChain` instead